src/BaGetter.Web/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -28,8 +30,40 @@ public static class IServiceCollectionExtensions
         services.AddTransient<IUrlGenerator, BaGetterUrlGenerator>();
 
         services.AddSingleton(ApplicationVersionHelper.GetVersion());
-        services.AddBaGetterApplication(configureAction);
+
+        var app = services.AddBaGetterApplication(configureAction);
+        app.AddNugetBasicHttpAuthentication();
+        app.AddNugetBasicHttpAuthorization();
```

Comment:
I'd prefer these calls in the BaGetter project so they are separate from the web app. Technically someone could add BaGetter as nuget feed to their application but not use the web app, in which case they'd have to discover these calls by other means to use them.

Reply:
Moved